pom.xml

```diff
@@ -110,21 +106,14 @@
     </scm>
 
     <distributionManagement>
-        <!--
-        NOTE: The properties snapshots.repository.url and releases.repository.url should be defined in your .m2/settings.xml file.
-        -->
         <snapshotRepository>
-            <id>snapshots</id>
-            <url>${snapshots.repository.url}</url>
+            <id>ion-snapshots</id>
+            <url>http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-snapshots/</url>
         </snapshotRepository>
         <repository>
-            <id>releases</id>
-            <url>${releases.repository.url}</url>
+            <id>ion-releases</id>
+            <url>http://nexus.phx.connexta.com:8081/nexus/content/repositories/ion-releases/</url>
         </repository>
```

Comment:
✏️ Can the `<repositories>` section be removed?

Reply:
IIRC the repositories and plugin repositories sections are explicitly configured to force maven to use https instead of http to connect to remote repositories.